paddle/framework/channel_test.cc

```diff
@@ -11,16 +11,58 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
+#include <gtest/gtest.h>
 
 #include "paddle/framework/channel.h"
+#include "threadpool.h"
 
-#include "gtest/gtest.h"
+using paddle::framework::Channel;
+using paddle::framework::MakeChannel;
+using paddle::framework::CloseChannel;
 
 TEST(Channel, MakeAndClose) {
-  using paddle::framework::Channel;
-  using paddle::framework::MakeChannel;
-  using paddle::framework::CloseChannel;
+  Channel<int>* ch = MakeChannel<int>(10);
+  CloseChannel(ch);
+}
 
+TEST(Channel, Buffered) {
```

Comment:
Do we need a test case for "unbuffered" channel too?

Reply:
The implement of `unbuffered` channel is empty.
https://github.com/PaddlePaddle/Paddle/blob/fb1a0dfb157032f3eb7cdabcb445a77656ef2498/paddle/framework/details/unbuffered_channel.h#L41-L48

Reply:
This [issue](https://github.com/PaddlePaddle/Paddle/issues/7937) is about `unbuffered` channel.